Menu.h

```diff
@@ -0,0 +1,49 @@
+#ifndef PELICULAS_MENU_H
+#define PELICULAS_MENU_H
+
+#include "Lista.h"
+#include <fstream>
+#include <sstream>
+
+using namespace std;
+
+class Menu{
+
+private:
+
+Lista<string> ListaPV;
+
+Lista<string> ListaPNV;
+
+Lista<string> ListaPR;
+
+int opcion;
+
+void iterarListaPeliculasVistas();
+
+void iterarListaPeliculasNoVistas();
+
+void iterarListaPeliculasRecomendadas();
+
+public:
+
+    //Constructor con parametro
+    //Pre-Condicion: El archivo debe existir y ser del tipo string, y debe existir una lista
+    //Post-Condicion: Se crea un objeto del tipo Menu
+
+    Menu(Lista<string> ListaPV,Lista<string> ListaPNV,Lista<string> ListaPR);
```

Comment:
```suggestion
    //Constructor con parametro
    //Pre-Condicion: Las listas deben existir
    //Post-Condicion: Se crea un objeto del tipo Menu
    Menu(Lista<Pelicula> peliculasVistas, Lista<Pelicula> peliculasNoVistas, Lista<Pelicula> peliculasRecomendadas);
```
La precondicion debería ser que las 3 listas existan. No entendí cuál es el archivo que debería existir.